examples/inline.yaml

```diff
@@ -131,7 +131,7 @@ content:
 
   - _target_: nbprint.ContentCode
     content: |
-      from perspective import PerspectiveWidget
+      from perspective.widget import PerspectiveWidget
```

Comment:
I did not realize this was supported. Will it properly show all the different perspective chart types?

Reply:
this is for `perspective>=3` support, but should work with either